packages/e2e-tests/scripts/buildkite_run_fe_tests.js

```diff
@@ -0,0 +1,62 @@
+const { execSync, exec } = require("child_process");
+const getSecret = require("./aws_secrets");
+
+function run_fe_tests(CHROME_BINARY_PATH) {
+  // Install node deps
+  execSync("npm i -g yarn", {
+    stdio: "inherit",
+  });
+
+  execSync("yarn install", {
+    stdio: "inherit",
+  });
+
+  execSync("cp .env.sample .env", {
+    stdio: "inherit",
+  });
+
+  // Start the webserver.
+  let webProc = exec("yarn dev", (error, stdout, stderr) => {
+    if (error) {
+      console.error(`exec error: ${error}`);
+      return;
+    }
+    console.error(`yarn dev stdout: ${stdout}`);
+    console.error(`yarn dev stderr: ${stderr}`);
+  });
+
+  // Wait a little, to let the yarn dev server start up.
+  execSync("sleep 5");
+
+  if (!process.env.HASURA_ADMIN_SECRET) {
+    process.env.HASURA_ADMIN_SECRET = getSecret("prod/hasura-admin-secret", "us-east-2");
+  }
+
+  process.env.RECORD_REPLAY_DISPATCH_SERVER = "wss://dispatch.replay.io";
+  process.env.REPLAY_BROWSER_BINARY_PATH = CHROME_BINARY_PATH;
+  process.env.RECORD_REPLAY_API_KEY = "rwk_b6mnJ00rI4pzlwkYmggmmmV1TVQXA0AUktRHoo4vGl9";
```

Comment:
Can't we set these in the `buildkite` pipeline settings?

Reply:
Create a new pipeline for mac arm, and create a new file that launches the tests under macos/arm64.  The pipeline can be adapted from this one and the ones in backend (backend/.buildkite/runtime-e2e-tests.yml) for running on arm boxes.

Reply:
This must be removed from here and also deleted on the backend.